app/src/components/SightingList.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+
+import * as apiClient from "../apiClient";
+
+import AnimalCard from "./AnimalCard";
+
+const SightingList = ({ sightings, individuals }) => {
+  const [selectedIndividual, setSelectedIndividual] = React.useState("");
+  const [individualDetails, setIndividualDetails] = React.useState({});
+  const [displayHealthyOnly, setDisplayHealthyOnly] = React.useState(false);
+  const [nicknameFilter, setNicknameFilter] = React.useState("");
+
+  const loadIndividual = React.useCallback(
+    () =>
+      apiClient.getIndividual(selectedIndividual).then(setIndividualDetails),
+    [selectedIndividual],
+  );
+
+  React.useEffect(() => {
+    selectedIndividual !== undefined && loadIndividual();
+  }, [selectedIndividual, loadIndividual]);
+
+  const healthyFilteredSightings = displayHealthyOnly
+    ? sightings.filter((sighting) => sighting.healthy)
+    : sightings;
+
+  const nicknameList = individuals.map((individual) => (
+    <option key={individual.id} value={individual.nickname}>
+      {individual.nickname}
+    </option>
+  ));
+
+  const filteredSightings = nicknameFilter
+    ? healthyFilteredSightings.filter(
+        (sighting) => sighting.nickname === nicknameFilter,
+      )
+    : healthyFilteredSightings;
+
+  return (
+    <section>
+      <h2>List of Sightings</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>Time Seen</th>
+            <th>
+              Nickname
+              <select
+                id="nickname"
+                name="nickname"
+                onBlur={(e) => setNicknameFilter(e.target.value)}
```

Comment:
I used `onChange` initially and got this error:
![Screen Shot 2021-09-17 at 2 14 53 PM](https://user-images.githubusercontent.com/11522217/133848481-83c1dd56-c6d9-429e-ae6a-f187953a38ef.png)
I did a little research and came across many discussions around onBlur vs. onChange for `<select>` tag as per eslint's (previous) requirements. eslint used to require onBlur to be used in place of onChange because of accessibility reasons but it seems that they [deprecated it](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md). However, I am still getting this error when I use `onChange`. I will try to look for a solution for it.

Reply:
> [suggest] I think it's better UX to do `onChange` instead of `onBlur`.

I used `onChange` initially and got this error:
![Screen Shot 2021-09-17 at 2 14 53 PM](https://user-images.githubusercontent.com/11522217/133848481-83c1dd56-c6d9-429e-ae6a-f187953a38ef.png)
I did a little research and came across many [discussions](https://github.com/mozilla/ensemble/issues/67) around onBlur vs. onChange for `<select>` tag as per eslint's (previous) requirements. eslint used to require onBlur to be used in place of onChange because of accessibility reasons but it seems that they [deprecated it](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md). However, I am still getting this error when I use `onChange`. I will try to look for a solution for it.

Reply:
> [suggest] I think it's better UX to do `onChange` instead of `onBlur`.

I used `onChange` initially and got this error:
![Screen Shot 2021-09-17 at 2 14 53 PM](https://user-images.githubusercontent.com/11522217/133848481-83c1dd56-c6d9-429e-ae6a-f187953a38ef.png)
I did a little research and came across many [discussions](https://github.com/mozilla/ensemble/issues/67) around onBlur vs. onChange for `<select>` tag as per eslint's (previous) requirements. eslint used to require onBlur to be used in place of onChange because of accessibility reasons but it seems that they [deprecated it](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md). However, I am still getting this error when I use `onChange`. I will try to look for a solution for it.

Reply:
You can just ask eslint to ignore just that line: https://eslint.org/docs/user-guide/configuring/rules#disabling-rules

Reply:
I've disabled this rule in the template: https://github.com/gsong/express-react-project-starter/commit/44314cb4cfb5e0f3afbd8255ba6afbed47e188cd